### ISSUE TITLE
A11y_mssql for VSCode_Results Tab: In high contrast mode, Color contrast ratio for "CTRL+ALT+R" is 4.4:1 which is less than  7:1

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/vscode-mssql#1234]
GitHubTags: #A11y_mssql for VSCode; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #Color Contrast; #MAS1.4.3; #mssql for VSCode

**Environment:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.17763 Build 17763)

**MAS ID Violated:** MAS 1.4.3

**Note:**
Similarly Color contrast ratio also fails for "CTRL+ALT+Y" which is present at Messages dropdown

**Repro Steps:**
1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write  code (SELECT *
FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query""  "
5. Change the color theme to high contrast(Files-->Preferences-->Color Theme-->High Contrast)
6. Verify color contrast ratio for "CTRL+ALT+R" is >= 7:1 or not

**Actual Result:**
 Color contrast ratio for "CTRL+ALT+R" is 4.4:1 which is less than  7:1

**Expected Result:**
Color contrast ratio for "CTRL+ALT+R" should be greater than or equal to  7:1

**User Impact:**
If color contrast ratio is less than 7:1 in high contrast theme then low vision users finds difficult in seeing the content present on the screen

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachments:**
![Bug_1234_color contrast](https://user-images.githubusercontent.com/42638450/56735796-1b20f700-6784-11e9-86b5-d6264ca9add5.png)

**Comment by geleems** [comment]
Reopened this for validation with test.

**Comment by kenvanhyning** [comment]
Cleaning up fixed accessibility issues.

**Comment by fsteffi** [comment]
#closed;
![image](https://github.com/microsoft/vscode-mssql/assets/24580570/6fe7331c-274b-4f07-b511-5d43d1570d73)
Version: 1.79.0-insider (user setup)
Commit: fa8eefda18bf099d82b58453f8fe5d3010196fa0
Date: 2023-05-31T03:06:45.096Z
Electron: 22.5.4
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370
